pkgs/applications/misc/ebook2cw/configfile.patch

```diff
@@ -0,0 +1,11 @@
+--- ebook2cw.c	2017-11-08 19:52:58.298131348 -0700
++++ ebook2cw.c	2017-11-08 19:53:02.588231067 -0700
+@@ -136,7 +136,7 @@
+ 	char isomap[256][4]; 		/* by these strings */
+ 	char utf8map[256][8];
+ 
+-	char configfile[1025];
++	char configfile[2048];
```

Comment:
I've asked upstream to cut a new version with this patch.

Reply:
Upstream says it's fixed in trunk, but that a new version won't be cut for several weeks.

Would it be better to switch from the release version (0.8.2) to a SVN revision number and drop the patch, or leave as-is and wait for a new release?

Reply:
Never mind -- we can't switch to an SVN revision number until #31441 is fixed.